Dockerfile

```diff
@@ -1,24 +1,39 @@
-FROM ubuntu:trusty 
-
+FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
-# Use baseimage-docker's init system. 
-CMD ["/sbin/my_init"]
+ENV LAST_REFRESHED 2016-03-22
+ENV SYNCANO_APIROOT https://api.syncano.io/
```

Comment:
maybe move this ENV a little down as this is a non-common part (while rest at top and bottom is common if we would want to unify all images automatically)